common/models/user.js

```diff
@@ -475,13 +602,13 @@ module.exports = function(User) {
    */
 
   User.prototype.verify = function(options, fn) {
+    options = options || this.getVerifyOptions();
```

Comment:
+1 for renaming `options` to `verifyOptions` eventually.

We should eventually fix this method to support context-based `options` argument like other methods already do, and then `User.verify` should pass `options` down to this method too.

Ideally, the change fixing `User.prototype.verify` to support `options` argument should be implemented in a standalone pull request, accompanied by new tests, and landed before we proceed with this patch #3314.